### DISCUSSION TITLE
How can I pre-fill or pre-check records from the table ?

### DISCUSSION THREAD

**Discussion by webdesign7** [filamentphp/filament#9482] [Help]
I know records in the Filament table can be selectable via checkboxes ( when there are configured bulk actions )  

So I am looking to pass somehow an array `selectedRecords` to the table somehow ( in my case .. I am using table widget ) so that those records become pre-checked when user loads the table. 

I noticed currently the state of the checkboxes are only handled by alpine but not by livewire 
vendor/filament/tables/resources/views/index.blade.php:1035
x-model="selectedRecords" 

is that possible ? 
Thanks

**Comment by sinnbeck** [comment]
I dont see that one line 1035 ?
https://github.com/filamentphp/tables/blob/3.x/resources/views/index.blade.php#L1035

Its here https://github.com/filamentphp/tables/blob/3.x/resources/views/index.blade.php#L523

Anyways. It does see to use livewire to bind it here: https://github.com/filamentphp/tables/blob/3.x/resources/js/components/table.js#L16

So you should be able to toggle them via livewire

**Comment by elmudometal** [comment]
were you able to solve this problem?
